templates/_base/footer.html

```diff
@@ -25,7 +25,6 @@
   </div>
 </footer>
 
-<script src="/static/js/modules/global-nav.js"></script>
-<script>canonicalGlobalNav.createNav({maxWidth: '66rem'});</script>
```

Comment:
You no longer appear to be initialising the global nav anywhere - or did I miss something?

Reply:
Doing it inside the Script directly :)